frontend/webpack.config.ts

```diff
@@ -207,7 +206,7 @@ const config: Configuration = {
     new CopyWebpackPlugin([{ from: './packages/knative-plugin/locales', to: 'locales' }]),
     new CopyWebpackPlugin([{ from: './packages/container-security/locales', to: 'locales' }]),
     new MomentLocalesPlugin({
-      localesToKeep: Object.keys(SUPPORTED_LOCALES).map((key) => (key === 'zh' ? 'zh-cn' : key)),
+      localesToKeep: ['en', 'ja'].map((key) => (key === 'zh' ? 'zh-cn' : key)),
```

Comment:
```suggestion
      localesToKeep: ['en', 'ja'],
```

And when we get Chinese, we can just add `zh-cn` to the array without needing `map`